File: CSharp/Route.cs
using System;
using System.Collections.Generic;

namespace CSharp
{
    class Route
    {
        private int id;
        private List<double> start = new List<double>();
        private List<double> end = new List<double>(); 

        public Route(int id, List<double> start, List<double> end)
        {
            this.id = id;
            this.start = start;
            this.end = end;
        }
    }
}

File: Java/src/Route.java
import java.util.ArrayList;

public class Route {
    private Integer id;
    private ArrayList<Double> start = new ArrayList<Double>();
    private ArrayList<Double>  end = new ArrayList<Double>();

    public Route(Integer id, ArrayList<Double> start, ArrayList<Double> end) {
        this.id = id;
        this.start = start;
        this.end = end;
    }
}
